frontend/src/app/features/work-packages/routing/partitioned-query-space-page/partitioned-query-space-page.component.ts

```diff
@@ -175,6 +181,14 @@ export class PartitionedQuerySpacePageComponent extends WorkPackagesViewBase imp
       });
   }
 
+  breadcrumbItems() {}
```

Comment:
I think we should enforce that this is implemented in the subclasses of the partitioned query space
```suggestion
  breadcrumbItems() {
    throw new Error('Not implemented');
  }
```